restapi/src/main/java/io/stargate/web/docsapi/models/CollectionUpgradeType.java

```diff
@@ -0,0 +1,5 @@
+package io.stargate.web.docsapi.models;
+
+public enum CollectionUpgradeType {
```

Comment:
Curious, what else are you thinking about putting in here?

Reply:
I have no idea xD I just made it because maybe in the future we decide the collection schema could be different, etc. and this would allow for it